keras/backend/jax/linalg.py

```diff
@@ -0,0 +1,72 @@
+import jax
+import jax.numpy as jnp
+import jax.scipy as jsp
+
+from keras.backend import config
+from keras.backend import standardize_dtype
+from keras.backend.common import dtypes
+from keras.backend.jax.core import cast
+from keras.backend.jax.core import convert_to_tensor
+
+
+def cholesky(a):
+    out = jnp.linalg.cholesky(a)
+    if jnp.any(jnp.isnan(out)):
+        raise ValueError(
+            "Cholesky decomposition failed. "
+            "The input might not be a valid positive definite matrix."
+        )
+    return out
+
+
+def det(a):
```

Comment:
Call `convert_to_tensor` on the input at the start of the ops

Reply:
I am doing so at the start of `_det`. Should I move this inside `det` instead?